openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -452,6 +453,9 @@ class CoursewareInformation(RetrieveAPIView):
         * user_has_passing_grade: Whether or not the effective user's grade is equal to or above the courses minimum
             passing grade
         * course_exit_page_is_active: Flag for the learning mfe on whether or not the course exit page should display
+        * can_view_certificate: Flag to determine whether or not the learner can view their course certificate. We pass
+            This seperately from certificate_data because certificate_data is only populated for passing students who
```

Comment:
The tiniest nit ever:
```suggestion
            this seperately from certificate_data because certificate_data is only populated for passing students who
```